spec/src/main/asciidoc/core/scopescontexts.asciidoc

```diff
@@ -491,6 +491,16 @@ An event with qualifier `@Initialized(RequestScoped.class)` is synchronously fir
 An event with qualifier `@BeforeDestroyed(RequestScoped.class)` is synchronously fired when the request context is about to be destroyed, i.e. before the actual destruction.
 An event with qualifier `@Destroyed(RequestScoped.class)` is synchronously fired when the request context is destroyed, i.e. after the actual destruction.
 
+The request context is active:
+
+* during any asynchronous invocation of an event observer,
+* during `@PostConstruct` callback of any bean.
+
+The request context is destroyed:
+
+* after the asynchronous invocation of an event observer,
```

Comment:
Agreed with both - but just to point out this is existing wording.  I can change it.